libraries/ashrae/guideline36/4.10-chilled-water-plant.ttl

```diff
@@ -0,0 +1,1221 @@
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Collection_Basin_Water_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] .
+
+# 4.10.7 Condenser Water Loop
+# The following points apply for water-cooled plants. Delete otherwise.
+
+:condenser-water-loop a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Condenser Water Loop" ;
+    sh:class brick:Chilled_Water_Loop ;
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [
+            sh:or (
+                sh:node components:constant-speed-pump
+                sh:node components:variable-speed-pump
+            )
+        ] ;
+        sh:qualifiedMinCount 1
+    ] ;
+    # required, TODO see comment
+    sh:property [
+        rdfs:label "Common CW supply temperature from towers" ;
+        rdfs:comment "If controlling cooling towers to maintain CWST (see 5.20.12.2), CWST sensor must be hardwired to the same controller as the cooling towers." ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Leaving_Condenser_Water_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    # required, TODO see comment
+    sh:property [
+        rdfs:label "Common CW return temperature to towers" ;
+        rdfs:comment "If controlling cooling towers to maintain CWRT (as opposed to CWST, see 5.20.12.2), CWRT sensor must be hardwired to the same controller as the cooling towers." ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Entering_Condenser_Water_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] ;
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [
+            sh:class brick:Makeup_Water_Valve ;
+            # required
+            sh:property [
+                rdfs:label "Makeup water valve" ;
+                sh:path brick:hasPoint ;
+                sh:qualifiedValueShape [ sh:class brick:On_Off_Command ] ;
+                sh:qualifiedMinCount 1 ;
+                sh:qualifiedMaxCount 1
+            ]
+        ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1
+    ] .
+
+# 4.10.8 Waterside Economizer
+# Retain the following points if the plant has a waterside economizer. Delete otherwise. “R” in this section should
+# be interpreted as required for waterside economizer operation.
+
+:chilled-water-loop-without-wse a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Condenser Water Loop without Waterside Economizer" ;
+    sh:node :chilled-water-loop .
+
+:wse-loop a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Waterside Economizer Loop" ;
+    sh:class brick:Chilled_Water_Loop ;
+    sh:property [
+        rdfs:label "Waterside Economizer" ;
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [
+            sh:class brick:Heat_Exchanger ;
```

Comment:
I agree with your comment “WSEs are HXs and the Economizer definition is vague”. Still, we call a damper an Economizer in the air-side economizer system, right?

Reply:
The [Economizer](https://brickschema.org/ontology/1.3/classes/Economizer) class doesn't have any children, which makes me think it could be moved or removed. Whereas the `Damper` class has a child class for `Economizer_Damper`, which makes me think the `Heat_Exchanger` class needs an `Economizer_Heat_Exchanger` child class.